video/decode/vd_lavc.c

```diff
@@ -696,6 +697,13 @@ static void init_avctx(struct mp_filter *vd)
         mp_set_avcodec_threads(vd->log, avctx, lavc_param->threads);
     }
 
+    if (lavc_param->dr == -1) {
+        bool wants_dr = false;
```

Comment:
another (minor) thing: if the vo doesn't have an answer for us we may want to default to true